api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java

```diff
@@ -127,13 +135,36 @@ public void findCriteriaMenu() {
                 .build());
     assertThat(
             controller
-                .findCriteriaMenu(WORKSPACE_NAMESPACE, WORKSPACE_ID, 0L)
+                .findCriteriaMenuOld(WORKSPACE_NAMESPACE, WORKSPACE_ID, 0L)
                 .getBody()
                 .getItems()
                 .get(0))
         .isEqualTo(cohortBuilderMapper.dbModelToClient(dbCriteriaMenu));
   }
 
+  //  TODO: Clean up name once feature flag enableStandardSourceDomains is removed
```

Comment:
Shouldn't the //TODO to cleanup be on the test calling findCriteriaMenuOld?

Reply:
This was initially to update the name of the method, but since i am now using old for existing way (no standard/source) i can update the method name and remove to do

Reply:
Ok so i think i have already updated it in the latest version and moved to do :)